gui/default/index.html

```diff
@@ -25,6 +25,8 @@
 </head>
 
 <body>
+  <script src="/syncthing/development/logbar.js"></script>
```

Comment:
Can we not include scripts with an absolute leading path? If someone serves syncthing from a sub-path, these will break.